Controller.cs

```diff
@@ -25,7 +25,68 @@ public static class Controller {
         public const int P16 = 16;
         public const int P19 = 19;
         public const int P20 = 20;
-        public const string BUILTIN_TEXT_LED = "led";
+
+        public const int LED = 21;
```

Comment:
Would it make sense to use negative numbers for the contacts that do not map to edge pins? That way if you ever have a device that uses pin 21, 22 etc. you can still accommodate that without having a gap in the constants that maps to default onboard devices.